internal/registry/registry.go

```diff
@@ -16,22 +16,21 @@
 
 package registry
 
-import "github.com/distribution/reference"
-
 const (
 	// IndexHostname is the index hostname, used for authentication and image search.
 	IndexHostname = "index.docker.io"
 	// IndexServer is used for user auth and image search
 	IndexServer = "https://index.docker.io/v1/"
 	// IndexName is the name of the index
 	IndexName = "docker.io"
+	// LegacyRegistry is the legacy name of the inde
```

Comment:
This is actually the canonical, only correct  hostname of the registry 🫠 
https://github.com/moby/moby/blob/2670796a010406b6b35aaff1cdd02f4309e79060/daemon/pkg/registry/config.go#L40-L46